### DISCUSSION TITLE
Is there any port for raspberry pi?

### DISCUSSION THREAD

**Discussion by marcelomanzo** [sparkle-project/Sparkle#2142] [Q&A] [answered]
I am huge fan of sparkly, I am wondering if is there any port for raspberry os?

**Comment by marcelomanzo** [comment]
or any recommended alternative

**Comment by zorgiepoo** [comment, marked as answer]
The Sparkle-like alternatives for other platforms I'm currently aware of are [WinSparkle](https://github.com/vslavik/winsparkle) and [NetSparkle](https://github.com/NetSparkleUpdater/NetSparkle). The latter claims to support Linux if the  raspberry pi runs a linux-like OS. Typically from what I hear software gets updated through package managers and app centers on linux though. That is all I really know.

**Reply by marcelomanzo** [reply]
I am familiar with the winsparkle it's windows based (dll).
I will check the netsparkle.

Thanks a lot!
